schema.js

```diff
@@ -15,6 +16,9 @@ module.exports = Joi.object().keys({
       import: Joi.array().items(Joi.object().keys({
         filename: Joi.string()
       }).requiredKeys('filename').unknown(true)),
+      download: Joi.array().items(Joi.object().keys({
+        sourceURL: Joi.string()
```

Comment:
If the source URLs are all `http` or `https`, then the Joi schema can be stricter: `Joi.string().uri({ scheme: /https?/ })`